### ISSUE TITLE
Ask for help with connection when installing speedtest-cli with wget, curl

### ISSUE THREAD

**Issue by NgHuuAn** [sivel/speedtest-cli#729]
I had used normal speedtest-cli before but recently we encountered an error. I did not hesitate to reinstall speedtest-cli but encountered an error

[root@server ~]# wget -O speedtest-cli https://raw.githubusercontent.com/sivel/speedtest-cli/master/speedtest.py
--2020-04-29 16:47:29--  https://raw.githubusercontent.com/sivel/speedtest-cli/master/speedtest.py
Resolving raw.githubusercontent.com (raw.githubusercontent.com)... 151.101.0.133, 151.101.64.133, 151.101.128.133, ...
Connecting to raw.githubusercontent.com (raw.githubusercontent.com)|151.101.0.133|:443... connected.
Unable to establish SSL connection.

The image below is the error that I encountered when running with curl, wget.

![image](https://user-images.githubusercontent.com/31845795/80626484-baa05600-8a78-11ea-87c0-12e2a1ddc9f2.png)

I have using wget, curl with some sources with https and there is no problem, hope to receive your feedback to solve this problem.

Thanks !

**Comment by mbonne** [comment]
what about: 
`curl -s https://raw.githubusercontent.com/sivel/speedtest-cli/master/speedtest.py | python -`

I really hope this git repo is secure, cos its very handy running script from here to test internet speeds, but aint nobody got time to check the script first start to finish before running it 😳

**Comment by NgHuuAn** [comment]
Yes, I can download the above command, but the downloaded file is an empty file.

This problem is unusual. I have reinstalled wget, curl, ca-certificates with the latest version corresponding to my operating system and temporary disable firewall but the problem is still here.

Through further testing, I found that there seemed to be a problem from my computer connecting SSL to raw.githubusercontent.com. Thanks for your feedback, if there is any problem solved I will update
